src/modules/collection/sagas.ts

```diff
@@ -153,8 +154,11 @@ export function* collectionSaga(builder: BuilderAPI, catalyst: CatalystClient) {
   function* handleFetchCollectionRequest(action: FetchCollectionRequestAction) {
     const { id } = action.payload
     try {
-      const collection: Collection = yield call(() => builder.fetchCollection(id))
+      const collection: Collection = yield call([builder, 'fetchCollection'], id)
       yield put(fetchCollectionSuccess(id, collection))
+      if (getCollectionType(collection) === CollectionType.THIRD_PARTY) {
+        yield put(fetchItemCurationsRequest(collection.id))
+      }
```

Comment:
I think it might be better to just listen to `FETCH_COLLECTION_SUCCESS` inside `modules/itemCuration/sagas` and fire this request there. This is so we keep consistent with how the rest of the modules interact with other modules' actions.

For an example, check:
`yield takeLatest(CONNECT_WALLET_SUCCESS, handleConnectWallet)`

on `modules/committee/sagas`